elements/pfe-readtime/README.md

```diff
@@ -0,0 +1,71 @@
+# PatternFly Element | Readtime element
+It takes in the word count of a given section and does a calculation on that number to return an estimated read time.
+
+## Usage
+
+```html
+    <!-- Default slot -->
+    <pfe-readtime for="readtime1">-minute readtime</pfe-readtime>
```

Comment:
I suspect this can be updated with examples utilizing the new slots we added: `read-string` and `read-string-less`